### ISSUE TITLE
X-scrolling rendering issue (not a bug)

### ISSUE THREAD

**Issue by rgavrilov** [DataTables/DataTables#548]
TLDR;
I have a page with a DataTable with enabled X-scrolling.
If I resize a page - the table gets resized as expected.
But if a vertical scroll bar gets added to the browser window - table does not detect that change, and adds h-scrollbar, even though there is plenty of space.

Details:
I have a page that loads components at the bottom dynamically, AFTER DT table has been initialized.
On initial page rendering page is short, and there is no vertical scroll bar and everything looks neat.

But when more components get added to the bottom - browser adds a vertical scroll bar, and reduces the page width by those 20px. As a result DT gets an X-scrollbar with only 20px of range of motion.

To demonstrate a problem, here is a possible solution:
use jquery.resize to detect page (or tables' container div) width change, and trigger table resizing.

I understand the issue isn't specific to DT, but I hope there is (or will be) a solution to address this.

**Comment by DataTables** [comment]
Can you link to a page showing the issue please.
